web_refresher/static/src/js/control_panel.esm.js

```diff
@@ -0,0 +1,11 @@
+/** @odoo-module **/
+/* Copyright 2022 Tecnativa - Alexandre D. Díaz
+ * Copyright 2022 Tecnativa - Carlos Roca
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
```

Comment:
shouldn't the link go to http://www.gnu.org/licenses/agpl?
otherwise it might not be clear which license you intend to use.
please check this for all added files.

Reply:
functional tests in chrome and firefox where OK